database/src/main/java/com/toggl/database/models/DatabaseUser.kt

```diff
@@ -12,7 +12,7 @@ import androidx.room.PrimaryKey
 )
 data class DatabaseUser(
     @PrimaryKey
-    val id: Long = 0,
+    val serverId: Long = 0,
```

Comment:
Do we really want to provide that default value here?
```suggestion
    val serverId: Long,
```